manifests/init.pp

```diff
@@ -75,6 +75,10 @@
 #
 # $rest_client_timeout:: Timeout for Katello rest API
 #
+# $pulp_worker_timeout:: The amount of time (in seconds) before considering a worker as missing. If Pulp's
+#                        mongo database has slow I/O, then setting a higher number may resolve issues where workers are
+#                        going missing incorrectly. Defaults to 30.
```

Comment:
The default in params.pp is 60. Not sure which we should default to but they should match :)